README.md

```diff
@@ -73,16 +72,15 @@ The above configuration is loaded taking into account the following priority ord
 
 1. Environment Variables.
 
-
-```
+```console
 ARGOCD_AUTH_TOKEN=ey...
 ARGOCD_SERVER_ADDR=argocd-server
 ARGOCD_INSECURE=true
 ```
 
+1. Files in the Config Directory (`/etc/prc-config/`).
```

Comment:
small thing, but `prc-config` doesn't line up with the `prcconfig` below.
And should we rename this `psc-config`?

Reply:
Fixed with a more explicit `applicationset-progressive-sync`